modules/gdscript/gdscript_parser.cpp

```diff
@@ -1745,7 +1746,8 @@ GDScriptParser::MatchBranchNode *GDScriptParser::parse_match_branch() {
 		push_error(R"(No pattern found for "match" branch.)");
 	}
 
-	consume(GDScriptTokenizer::Token::COLON, R"(Expected ":" after "match" patterns.)");
+	if (!consume(GDScriptTokenizer::Token::COLON, R"(Expected ":" after "match" patterns.)"))
+		return nullptr;
```

Comment:
Need to use braces on `if` statements to follow the style guidelines.

Reply:
Should be good now. Out of curiosity, why doesn't aren't there any checks for that?

Reply:
> Should be good now. Out of curiosity, why aren't there any checks for that?

Not sure, I think clang-format doesn't have a check for that and adding another tool just for this check is not ideal.